packages/react-ui-core/src/DateInput/DateInput.js

```diff
@@ -1,24 +1,33 @@
 import React, { Component } from 'react'
 import PropTypes from 'prop-types'
-import DatePicker from 'react-datepicker'
+import DatePicker from 'react-basic-datepicker'
 
 export default class DateInput extends Component {
+  static propTypes = {
+    selected: PropTypes.object,
+    onChange: PropTypes.func,
+    maxDate: PropTypes.object,
+    minDate: PropTypes.object,
+    placeholderText: PropTypes.string,
+    className: PropTypes.string,
```

Comment:
`className` prop doesn't seem to be used at all (not passed through to `<DatePicker>`